cmake/features.cmake

```diff
@@ -128,6 +126,8 @@ ie_dependent_option(NGRAPH_ONNX_IMPORT_ENABLE "Enable ONNX importer" ON "protoc_
 ie_dependent_option(NGRAPH_PDPD_FRONTEND_ENABLE "Enable PaddlePaddle FrontEnd" ON "protoc_available" OFF)
 ie_dependent_option(NGRAPH_USE_PROTOBUF_LITE "Compiles and links with protobuf-lite" OFF
     "NGRAPH_ONNX_IMPORT_ENABLE OR NGRAPH_PDPD_FRONTEND_ENABLE" OFF)
+ie_dependent_option(NGRAPH_USE_SYSTEM_PROTOBUF "Use system protobuf" OFF
```

Comment:
Is this really needed?

Reply:
yes, it was deleted by mistake here https://github.com/openvinotoolkit/openvino/commit/bc3642538190a622265560be6d88096a18d8a842#diff-3d2b555bea347adb474b05f13a23deef8ff07c2fb21bc167a5f90a09ea16b796L28